backend/src/test/java/gov/cdc/usds/simplereport/db/repository/TestEventRepositoryTest.java

```diff
@@ -108,18 +109,20 @@ void testLatestTestEventForPerson() {
     TestOrder firstOrder =
         _dataFactory.createCompletedTestOrder(patient, place, TestResult.POSITIVE);
     var firstResults = firstOrder.getResults();
-    var firstEvent = new TestEvent(firstOrder, false, firstResults);
+    var firstEvent = new TestEvent(firstOrder, false);
+    _repo.save(firstEvent);
     firstResults.forEach(result -> result.setTestEvent(firstEvent));
```

Comment:
could we use the new ResultService.addResultsToTestEvent here and in other tests in this file?

Reply:
done